components/brave_wallet_ui/utils/asset-utils.ts

```diff
@@ -169,7 +169,7 @@ export const sortNativeAndAndBatAssetsToTop = (
 
 export type GetBlockchainTokenIdArg = Pick<
   BraveWallet.BlockchainToken,
-  'coin' | 'chainId' | 'contractAddress' | 'isErc721' | 'tokenId' | 'isNft'
+  'coin' | 'chainId' | 'contractAddress' | 'isErc721' | 'tokenId' | 'isNft' | 'isCompressed'
```

Comment:
is this ('isCompressed') required?

Reply:
Not 100% sure what you mean by required, but I think it probably ought to be required, at least for SOL NFTs, because it is necessary to determine which set of APIs to use for transfer and fetch metadata

Reply:
Gotcha - isCompressed doesn't need to be a part of that ID